contrib/opencensus-ext-fastapi/setup.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2022, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from setuptools import find_packages, setup
+
+from version import __version__
+
+setup(
+    name='opencensus-ext-fastapi',
+    version=__version__,  # noqa
+    author='Taishi Ikai',
+    author_email='ikai1026@gmail.com',
```

Comment:
Please use OpenCensus authors as in https://github.com/census-instrumentation/opencensus-python/blob/b85e4767edcd6524515d263853247335f9ee69f4/contrib/opencensus-ext-stackdriver/setup.py#L22-L23

Reply:
https://github.com/census-instrumentation/opencensus-python/pull/1124/commits/bcffefb8dc01a8c6960d23c44c7185682547cfb8